IBPSA/Airflow/Multizone/EffectiveAirLeakageArea.mo

```diff
@@ -14,6 +14,7 @@ model EffectiveAirLeakageArea "Effective air leakage area"
     annotation (Dialog(group="Rating conditions"));
 
   parameter Modelica.SIunits.Area L(min=0) "Effective leakage area";
+  Modelica.SIunits.Velocity v(nominal=1) "Average velocity";
```

Comment:
You added a real variable but it was already used on line 19, so it existed already?

Reply:
The average velocity variable is specific to the type of flow element and was moved to a higher level.  Only EffectiveAirLeakageArea and Orifice use this variable.